xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -1398,7 +1398,14 @@ bool CRenderManager::GetStats(int &lateframes, double &pts, int &queued, int &di
 
 void CRenderManager::CheckEnableClockSync()
 {
-  if (fabs(m_fps - g_graphicsContext.GetFPS()) < 0.01)
+  // refresh rate can be a multiple of video fps
+  double diff;
+  if (g_graphicsContext.GetFPS() >= m_fps)
+    diff = fmod(g_graphicsContext.GetFPS(), m_fps);
```

Comment:
fmod(59.94, 30.0) =  29.94

An additional condition if wanted could be || abs(m_fps - diff) < threshold